src/commands/cmd_bit.cc

```diff
@@ -143,6 +143,7 @@ class CommandBitCount : public Commander {
  private:
   int64_t start_ = 0;
   int64_t stop_ = -1;
+  bool is_bit_ = false;
```

Comment:
Other commands have different meanings of `bit_`, which may cause misunderstanding. 
Can you change it to another name? Such as `type` or `flag`? If you can change it to an enum, it is better! 

https://github.com/kay011/kvrocks/blob/64ba5f9e72e85429fe625878da8d6442ba0ff475/src/commands/cmd_bit.cc#L91
https://github.com/kay011/kvrocks/blob/64ba5f9e72e85429fe625878da8d6442ba0ff475/src/commands/cmd_bit.cc#L201

Reply:
I think something like `position_is_bit_` would be better